cloud/core/Cargo.toml

```diff
@@ -9,8 +9,28 @@ authors = ["Scuffle <opensource@scuffle.cloud>"]
 readme = "README.md"
 publish = false
 
+[[bin]]
+name = "scufflecloud-core"
+path = "src/bin/standalone.rs"
+
 [dependencies]
+anyhow = "1.0.98"
+async-trait = "0.1.88"
+diesel = { features = ["postgres"], version = "2.2.10" } # libpq-dev has to be installed
```

Comment:
hmmmmmmmmmmmmmm, this is wrong.

I am pretty sure dont use this feature flag.

Reply:
https://github.com/ScuffleCloud/brawl/blob/d2035c2a1f87d5b5411abde1aa6221e0e5d40be1/server/Cargo.toml#L38-L39

See this